src/main.ts

```diff
@@ -29,10 +29,17 @@ export default class LightSheet {
   }
 
   initializeData() {
+    // Create header row and add headers
+    const headerData = Array.from(
+      { length: this.options.data[0].length + 1 }, // Adding 1 for the row number column
+      (_, i) => (i === 0 ? "" : String.fromCharCode(64 + i)), // A for the first column
+    );
```

Comment:
this wont hold after column name "Z". 

This link might help to do this better: https://stackoverflow.com/a/8241071

Reply:
new code is
    // Create header row and add headers
    const headerData = Array.from(
      { length: this.options.data[0].length + 1 }, // Adding 1 for the row number column
      (_, i) => (i === 0 ? "" : this.generateRowLabel(i)), // Generating row labels
    );
    this.ui.addHeader(headerData);


  generateRowLabel(rowIndex: number) {
    let label = "";
    const alphabet = "ABCDEFGHIJKLMNOPQRSTUVWXYZ";
    while (rowIndex > 0) {
      rowIndex--; // Adjust index to start from 0
      label = alphabet[rowIndex % 26] + label;
      rowIndex = Math.floor(rowIndex / 26);
    }
    return label || "A"; // Return "A" if index is 0
  }